e2e-tests/src/tests/studies/ourhealth/preenroll-response.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from '@playwright/test'
+import { test } from 'lib/fixtures/ourhealth-fixture'
+import { Study } from 'src/data/constants-en'
+import { PreEnrollResponse } from 'src/models/preenroll-response'
+import { goToStudyEligibility, waitForResponse } from 'tests/e2e-utils'
+
+test.describe('POST /preEnroll request', () => {
```

Comment:
Network response parsing or validation test example. 

Test log looks similar to this screenshot.

<img width="991" alt="Screenshot 2024-03-26 at 4 16 10 PM" src="https://github.com/broadinstitute/juniper/assets/35533885/54e94994-b7c7-4e0a-a53b-dbf176f97d38">